src/orange/Types.hh

```diff
@@ -28,6 +28,9 @@ using SurfaceId = OpaqueId<struct Surface>;
 //! Identifier for a geometry volume
 using VolumeId = OpaqueId<struct Volume>;
 
+//! Identifier for a BIHNode objects
+using BIHNodeId = OpaqueId<struct BIHNode>;
```

Comment:
I think this belongs in `OrangeTypes` since it's not for geometry usage in general. Also make sure to keep the list alphabetized when you move it.